cargo-dist/src/lib.rs

```diff
@@ -597,10 +597,15 @@ fn really_zip_dir(src_path: &Utf8Path, dest_path: &Utf8Path) -> Result<()> {
 
 /// Arguments for `cargo dist init` ([`do_init`][])
 #[derive(Debug)]
-pub struct InitArgs {}
+pub struct InitArgs {
```

Comment:
for lib version of this i would either drop it or i would turn this into some sort of config struct that we expect folks to hand us- i think having interaction in this part is a bit odd

Reply:
(and move the interaction to a different place) otherwise it is very confusing to read tbh, sorta a "uncanny valley" space